app/controllers/replacement_document_validation_requests_controller.rb

```diff
@@ -31,6 +31,11 @@ def send_validation_request_email
     ).deliver_now
   end
 
+  def email_and_timestamp
+    send_validation_request_email
+    @replacement_document_validation_request.update!(notified_at: Time.zone.now)
+  end
```

Comment:
seems like a lot of code repetition - could we not pop this method into ValidationRequestsController and make these controllers inherit from it?

Reply:
(same with the send_validation_request email, we could pop both into that controller theoretically and just call it with an argument or validation_request type, it seems the code is kind of exactly the same except for the request type?)

Reply:
Yes, good idea